docs/rules/object-shorthand.md

```diff
@@ -0,0 +1,27 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/object-shorthand
+description: require or disallow method and property shorthand syntax for object literals
```

Comment:
Maybe to make this clear right from the short description?

```suggestion
description: require or disallow method and property shorthand syntax for object literals in the template
```

Reply:
Your point is correct, but it's automatically generated, so it's difficult to fix now :sweat_smile:. We need to copy and replace the description of all extended rules.

Reply:
Ahh I see. I didn't know that the descriptions of extended core rules are also copied. Does it make sense to always append " in `<template>`"? Because that's what the extended core rules always do?

Reply:
Anyway, that should not block this PR. I'll create a new issue for that.

Reply:
Done in #1764.